.gitlab-ci.yml

```diff
@@ -89,3 +89,75 @@ deploy_to_reliability_env:
     UPSTREAM_PACKAGE_JOB: build
     UPSTREAM_PROJECT_NAME: $CI_PROJECT_NAME
     FORCE_TRIGGER: $DEPLOY_TO_REL_ENV
+
+deploy_to_docker_registries:
+  stage: deploy
+  rules:
+    - if: '$POPULATE_CACHE'
+      when: never
+    - if: '$CI_COMMIT_TAG =~ /^v.*/'
+      when: on_success
+    - when: manual
+      allow_failure: true
+  trigger:
+    project: DataDog/public-images
+    branch: main
+    strategy: depend
+  variables:
+    IMG_SOURCES: ghcr.io/datadog/dd-trace-dotnet/dd-lib-dotnet-init:$CI_COMMIT_SHA
+    IMG_DESTINATIONS: dd-lib-dotnet-init:$CI_COMMIT_TAG
+    IMG_SIGNING: "false"
+
+deploy_musl_tag_to_docker_registries:
+  stage: deploy
+  rules:
+    - if: '$POPULATE_CACHE'
+      when: never
+    - if: '$CI_COMMIT_TAG =~ /^v.*/'
+      when: on_success
+    - when: manual
+      allow_failure: true
+  trigger:
+    project: DataDog/public-images
+    branch: main
+    strategy: depend
+  variables:
+    IMG_SOURCES: ghcr.io/datadog/dd-trace-dotnet/dd-lib-dotnet-init:$CI_COMMIT_SHA-musl
+    IMG_DESTINATIONS: dd-lib-dotnet-init:$CI_COMMIT_TAG-musl
+    IMG_SIGNING: "false"
+
+deploy_latest_tag_to_docker_registries:
+  stage: deploy
+  rules:
+    - if: '$POPULATE_CACHE'
+      when: never
+    - if: '$CI_COMMIT_TAG =~ /^v.*/'
+      when: on_success
+    - when: manual
+      allow_failure: true
+  trigger:
+    project: DataDog/public-images
+    branch: main
+    strategy: depend
+  variables:
+    IMG_SOURCES: ghcr.io/datadog/dd-trace-dotnet/dd-lib-dotnet-init:$CI_COMMIT_SHA
+    IMG_DESTINATIONS: dd-lib-dotnet-init:latest
```

Comment:
I think it hasn't been discussed and not done by other teams. Couldn't we add a `v2-latest` tag in addition to latest. Because as of now, our customers can put one version, or latest. But if we deliver a new major, we don't want them to use latest, don't we?

Reply:
This is a good idea. What do you think of keeping the existing tags the way they are (just vX.Y.Z) and we come back to this discussion and possibly add this at a later time?

Reply:
> This is a good idea. What do you think of keeping the existing tags the way they are (just vX.Y.Z) and we come back to this discussion and possibly add this at a later time?

I think that's fine. Though arguably it would be better to decide on an approach _now_ before customers start using it. That way all the _initial_ implementations they use will be the "safe" version if they want it (`v2`). Otherwise, if they add it later, _every_ customer needs to update their YAML. 

It should probably be a cross team thing though if we do, all tracer languages should have that IMO

Reply:
Keep in mind Python, Node, and Java are already GA :laughcry: